tests/fv/fv_test.go

```diff
@@ -74,7 +59,7 @@ var _ = Describe("PolicyController", func() {
 		kfconfigfile, err := ioutil.TempFile("", "ginkgo-policycontroller")
 		Expect(err).NotTo(HaveOccurred())
 		defer os.Remove(kfconfigfile.Name())
-		data := fmt.Sprintf(kubeconfigTemplate, apiserver.IP)
+		data := fmt.Sprintf(KubeconfigTemplate, apiserver.IP)
```

Comment:
Just noticed that this is imported with a `.` import - would be good to make this a regular import so it's easier to tell where it's coming from.